pages/api/mindmaps/[key]/index.js

```diff
@@ -4,6 +4,12 @@ import { rg2cy } from '../../../../utils/cyHelpers'
 import { chain } from 'lodash'
 
 const { db } = require('../../../../utils/arangoWrapper')
+const updateMindMap = async (database, mindmapKey, name) => {
+  return database.query({
```

Comment:
This needs to go through RecallGraph's API instead of a direct DB update. That way, RG can track change history for this event.

Please see https://github.com/adityamukho/hivemind/blob/dbdda9325c2e35348ef5716dad280d60e01c8344/pages/api/nodes/index.js#L61 for an example of how to do this,